indexer/src/types/transaction.ts

```diff
@@ -400,6 +397,19 @@ function addSignature(
         r,
         s,
       });
+      const txHash = "0x" + Array.from(legacyTx.hash())
+        .map((b) => b.toString(16).padStart(2, "0"))
+        .join("");
+
+      if (
+        v < 35 && Number(v) !== 27 && Number(v) !== 28
+      ) {
+        throw new Error(
+          `Legacy txs need either v = 27/28 or v >= 37 (EIP-155 replay protection), got v = ${v}`,
```

Comment:
`v >= 35`

Reply:
I mean in the error message, we have v>=37, instead of v>=35

Reply:
aaah you're right